Scripts/Commons/common.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+#####################
+#     CONSTANTS     #
+#####################
+
+readonly ERROR_HOMEBREW_NOT_INSTALLED=$(cat << MSG
+error: Homebrew not installed. See https://brew.sh/ for installation instructions.
+MSG
+)
+
+readonly ERROR_SWIFTFORMAT_CMD_MISSING=$(cat << MSG
+error: 'swift-format' command not found.
+MSG
+)
+
+#####################
+#     FUNCTIONS     #
+#####################
+
+function command_exists() {
```

Comment:
The `command_exists` function relies on `command` builtin rather than `which` external binary. When programming in Bash, and many other Unix shells, it is generally recommended to use `command` with `-v` option when checking for command existence, because this way you avoid a dependency on tool that is outside your shell. The function simply wraps it all up, so that we don't need to remember what option to use with `command` builtin and to `&>/dev/null` each time. I would recommend to leave it as is.